spec/helpers/metric_chart_helper_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
```

Comment:
I think it would be useful to have a test here with ```previous_metrics = [0, 0, 0, 0, 0, 0, 0, 0....``` as it might show a div by 0 when summed in ```difference_decrease_percentage```

Reply:
Have added.